README.md

```diff
@@ -57,3 +57,45 @@ Documentation is lacking at the moment, but contributions are welcome!
 1. Run `cd Example; bundle exec pod install`
 2. Open `Example/KanvasExample.xcworkspace` in Xcode
 3. Run the app on a device.
+
+## Bump version
+
+### Steps
+
+1. Open `Kanvas.podspec`
+2. Upgrade `spec.version`, please refer `Version naming rules` section below.
+3. `cd Example`
```

Comment:
❤️ Nice job including this. Personally I forget this a lot and end up not knowing what's gone wrong 😅